payment-service/src/main/java/com/programming/PaymentServiceApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.programming;
+
+import com.stripe.Stripe;
+import jakarta.annotation.PostConstruct;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class PaymentServiceApplication {
+
+    @PostConstruct
+    public void setup(){
+        Stripe.apiKey = "sk_test_51PH1GmSBC2UiAEfDhIBuAfMxKcNWNJ94epEVvjPCpmXgP6YGK1pJYfBvnMlGuDrLtbsKIAsFWtijdpZ0YwWih80L00ySMXcnSZ";
```

Comment:
remove credentials from here, do not use postconstruc to load key. handle it other way but secrets must not be in any properties or yaml file. THINK.